integration-tests/modules/__tests__/currency/admin/currency.spec.ts

```diff
@@ -60,9 +60,10 @@ medusaIntegrationTestRunner({
           adminHeaders
         )
 
-        expect(listResp.data.currencies).toHaveLength(1)
-        expect(listResp.data.currencies[0]).toEqual(
-          expect.objectContaining({ code: "zwl", name: "Zimbabwean Dollar" })
+        expect(listResp.data.currencies).toEqual(
```

Comment:
@adrien2p do you know what change in this PR is causing the module loaders on each test?

I had to add this fix to the test because earlier the module loader would be run once in a test and then all the data created in that loader will be deleted on any subsequent test. 

But now, its running on every test.

Reply:
weird, I didnt changed that part